### ISSUE TITLE
Section508.gov - Define Accessibility Criteria in Contracts

### ISSUE THREAD

**Issue by mgifford** [GSA/openacr#321]
In search for VPAT https://www.section508.gov/buy/define-accessibility-criteria/

In **Acceptance Criteria - Sample language** change:
> Before acceptance, the contractor shall provide an Accessibility Conformance Report (ACR) for each ICT item that is developed, updated, configured for the agency, and when product substitutions are offered. The ACR should be based on the latest version of OpenACR. If an OpenACR report is not available, a procurement team may decide to accept a legacy [Voluntary Product Accessibility Template (VPAT™)](https://www.section508.gov/sell/vpat) report. To be considered for award, an ACR must be submitted for each ICT Item, and must be completed according to the instructions provided by ITIC.

Todo: 
- [ ] Plain language
- [ ] Link to OpenACR

**Comment by mgifford** [comment]
See https://github.com/GSA/Section508.gov/pull/229
